server/src/api/modules/groups/services/FetchGroupDetailsService.ts

```diff
@@ -29,7 +30,8 @@ async function fetchGroupDetails(id: number): Promise<GroupDetails> {
     // Sort the members list by role
     memberships: group.memberships.sort(
       (a, b) => priorities.indexOf(b.role) - priorities.indexOf(a.role) || a.role.localeCompare(b.role)
-    )
+    ),
+    roleOrders: group.roleOrders
```

Comment:
Should we expose these? They're most useful just to sort the members list, idk if API consumers need them

Reply:
Scenario:

I have a discord bot for my clan, and we have a list members command. I want to sort my members in order of role.

I say keep it.

Reply:
Sounds good to me